2020-02-Spring/4_graphs/README.md

```diff
@@ -48,21 +48,62 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. Course Schedule
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/course-schedule/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Suppose that you are a college student who is planning on your course schedule.
+
+There are a total of `numCourses` courses that you have to take. 
+Each course is labeled from `0` to `numCourses - 1`.
+
+Some of these courses have prerequisites. 
+Suppose that course `1` is a prerequisite of course `0`. We can express this relationship with the following pair: `[0, 1]`
+
+Given the total number of courses `numCourses` and a list of prerequisite **pairs**, determine if it is possible for you to finish all of the courses.
 
 #### Example Input
 
-If the problem is simple enough, remove this section. TODO :bug:
+**Example 1**:
+
+Input: `numCourses = 2, prerequisites = [[1, 0]]`
+Output: `true`
+
+There are two courses that you must complete.
+To complete course 1, you must take course 0. 
+
+It is possible to complete all of thse courses.
+
+**Example 2**:
+
+Input: `numCourses = 2, prerequisites = [[1, 0]]`
```

Comment:
Is this correct? It looks the same as the last one and the example solution doesn't match up.